packages/pg-protocol/src/serializer.ts

```diff
@@ -197,40 +198,40 @@ type PortalOpts = {
   name?: string
 }
 
-const cstringMessage = (code: code, string: string): Buffer => {
+const cstringMessage = (code: code, string: string, encoding: TextEncoding): Buffer => {
```

Comment:
nit: would be nice to have ` = 'utf8'` as the default value here for consistency with the other functions.